screencapturepermissions.m

```diff
@@ -0,0 +1,36 @@
+#import <Foundation/Foundation.h>
+#include <node_api.h>
+
+static napi_value hasPermissions(napi_env env, napi_callback_info info) {
+  napi_status status;
+  bool hasPermissions;
+  napi_value hasPermissionsInt32;
+  napi_value ret;
+  CGDisplayStreamRef stream_ref;
+
+  stream_ref = CGDisplayStreamCreateWithDispatchQueue(CGMainDisplayID(), 1, 1, 'BGRA', nil, dispatch_get_main_queue(), ^(CGDisplayStreamFrameStatus status,
```

Comment:
Each parameter should be on separate lines.

Reply:
Done, thanks for the suggestion